pkg/lib/operation/template/local/test/operation.go

```diff
@@ -321,7 +324,40 @@ func runRemoteTest(testName string, tmpl *template.Template, repoFS filesystem.F
 		LogUntrackedPaths: true,
 		ChangeDescription: "",
 	}
-	return syncPush.Run(prjState, pushOpts, projectDeps)
+	err = syncPush.Run(prjState, pushOpts, projectDeps)
+	if err != nil {
+		return err
+	}
+
+	// Get mainConfig from template
+	tickets := storageapi.NewTicketProvider(d.Ctx(), testPrj.StorageAPIClient())
+	ctx := use.NewContext(d.Ctx(), tmpl.Reference(), tmpl.ObjectsRoot(), tmplInstID, branchKey, inputValues, tmpl.Inputs().InputsMap(), tickets)
+	tmplState, err := tmpl.LoadState(ctx, loadState.Options{
+		LoadLocalState:    true,
+		LoadRemoteState:   false,
+		IgnoreNotFoundErr: false,
+	})
+	if err != nil {
+		return err
+	}
+	mainConfig, err := tmplState.MainConfig()
+	if err != nil {
+		return err
+	}
+	spew.Dump(prjState.Manifest().All())
+
+	// Run the mainConfig job
+	queueClient := testPrj.JobsQueueAPIClient()
+	fmt.Printf(">>> PRJ %#v\n", testPrj.ID())
+	job, err := jobsqueueapi.CreateJobRequest(mainConfig.ComponentId, mainConfig.ConfigId).Send(ctx, queueClient)
```

Comment:
@webrouse prosím tě, mám tady asi divný problém s idčkem, který se dostal do `mainConfig.ConfigId`. Do šablony jsem to přidal dobře, ne? https://github.com/keboola/keboola-as-code/blob/c45c2ec915c0da9860e619223388c2e693e6172b/test/cli/template-test/remote/in/my-template/v0/src/manifest.jsonnet#L2-L5

Failuje mi request do Queue na tom že `Configuration 23030764 not found`. V projektu takovou opravdu nevidím a ani když si nechám vydumpovat `prjState.Manifest()` tak tam takový idčko není.

![CleanShot 2022-08-24 at 12 59 10](https://user-images.githubusercontent.com/215660/186402192-0d0a6633-d6ec-4ddf-9efa-f9c9494d4185.jpg)

Reply:
zatial som problem nenasiel, pozriem sa na to rano s cerstvou hlavou